### PR TITLE
Create tutorial.rst

### PR DESCRIPTION
**Pull request by Lekaveh**

### CODE REVIEW

**Review by keitaroyam — CHANGES_REQUESTED** [review]
Thank you @MartinMalyMM. I think json2restraints.py should be included in this pull request.

> Further, for an particular atom considering a particular coordination, the report consists of three sections: base, angles and pdb. All the stated ideal values have also their standard deviations denoted as std.

I would suggest to show a snippet of json file here.

> For instance consider the following case: an interaction of an iron (possessing a coordination of 4) with a carbon atom is reported in a JSON file.

Can you provide a PDB code that showcases this interaction? Plus, wouldn't "-c 3" eliminate the real interactions with coordination of four in the file (if exist)? Then it does not sound a good solution.

**Comment by MartinMalyMM** [comment]
I addressed the issues, added more examples and attached the relevant files and scripts.

File: tutorial/traingles.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    for mark1 in marks1:
+        cra1 = mark1.to_cra(st[0])
+        # print("  ", cra1.atom)
+        marks2 = ns.find_neighbors(cra1.atom, min_dist=0.1, max_dist=max_dist)
+        # print("    ", len(marks2), " connections")
+        marks2_filtered = []
+        for mark2 in marks2:
+            cra2 = mark2.to_cra(st[0])
+            dist_cra1_cra2 = cra1.atom.pos.dist(cra2.atom.pos)
+            dist_max = 1.3 * (cra1.atom.element.covalent_r + cra2.atom.element.covalent_r)
```
Comment (keitaroyam) [inline tutorial/traingles.py:42]:
```
            dist_max = 1.3 * (cra1.atom.element.covalent_r + cra2.atom.element.covalent_r)
```
factor?

File: tutorial/json2restraints.py [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+outputPdbPath = outputPrefix + ".pdb"
+
+# Delete LINK connection records of a type of MetalC
+if args.p:
+    st = gemmi.read_structure(stPath)
+    if not args.keep_links:
+        connections_kept = gemmi.ConnectionList()
+        for connection in st.connections:
+            if not (connection.type == gemmi.ConnectionType.MetalC or \
+                    connection.partner1.atom.element.is_metal or \
```
Comment (keitaroyam) [inline tutorial/json2restraints.py:60]: partner1/2 (gemmi.AtomAddress objects) don't have `atom` member.

**Pull request merged**